activerecord/lib/active_record/query_logs.rb

```diff
@@ -73,7 +74,9 @@ module QueryLogs
     mattr_accessor :tags, instance_accessor: false, default: [ :application ]
     mattr_accessor :prepend_comment, instance_accessor: false, default: false
     mattr_accessor :cache_query_log_tags, instance_accessor: false, default: false
+    mattr_accessor :tags_format, instance_accessor: false, default: :default
     thread_mattr_accessor :cached_comment, instance_accessor: false
+    thread_mattr_accessor :tags_formatter, instance_accessor: false
```

Comment:
What's the purpose of making the formatter per thread?

Reply:
Hmmm good question - I don't think this needs to be per thread. I updated this to use `mattr_accessor` instead, and was also able to remove the `ActiveSupport::ExecutionContext.after_change { ActiveRecord::QueryLogs.update_formatter }` hook below. I think the implementation is simpler now.